### PR TITLE
v3 search: Return package owners

### PR DESCRIPTION
**Pull request by CyberAndrii**
Added the "owners" array to search result, see https://docs.microsoft.com/en-us/nuget/api/search-query-service-resource#search-result

Closes https://github.com/NuGet/NuGetGallery/issues/5647

### CODE REVIEW

**Comment by dnfgituser** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/NuGet/NuGet.Jobs?pullRequest=980) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/NuGet/NuGet.Jobs?pullRequest=980) before we can accept your contribution.<br/><br/>:x: CyberAndrii [sign now](https://cla.dotnetfoundation.org/NuGet/NuGet.Jobs?pullRequest=980)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/NuGet/NuGet.Jobs?pullRequest=980) it.</sub>

File: tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchResponseBuilderFacts.cs [hunk 1]
```diff
@@ -884,6 +886,10 @@ public void ProducesExpectedResponse(bool useNewtonsoftJson)
       ""authors"": [
         ""Microsoft""
       ],
+      ""owners"": [
+        ""Microsoft"",
+        ""azure-sdk""
+      ],
```
Comment (loic-sharma) [inline tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchResponseBuilderFacts.cs:892]: Thanks for adding tests too! 🥇

**Comment by loic-sharma** [comment]
Thanks for opening this, great work!

Before moving forward, we'd like to check that the NuGet client and Visual Studio won't be impacted by this change. I'll work with our sister team to double-check this change doesn't break them. We'll merge this in once we get their approval

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# review when someone opens a pull request.
+# For more on how to customize the CODEOWNERS file - https://help.github.com/en/articles/about-code-owners
+*       @NuGet/gallery-team
```
Comment (loic-sharma) [inline .github/CODEOWNERS:5]: Hi I rebased your changes onto the `dev` branch (we merge new changes to the `dev` branch, and later merge the `dev` branch to `main` once ready for deployment).

This `CODEOWNERS` file was added to the `main` branch, could you update your branch to remove this file?

**Comment by joelverhagen** [comment]
Thanks for this PR @CyberAndrii! I am going to merge this into a feature branch and add some things one top: integration tests and a feature flag to allow us to quickly turn this behavior on and off.

**Review by joelverhagen — APPROVED** [review]

**Comment by joelverhagen** [comment]
This change is shipped to PROD. Thanks again for your help @CyberAndrii!

**Pull request merged**